src/rocker/templates/bashrc_extensions_user_snippet.Dockerfile.em

```diff
@@ -0,0 +1,6 @@
+WORKDIR @home_dir
```

Comment:
We need to be careful about setting this WORKDIR as it sideeffects downstream snippets.